src/components/SearchBar/SearchBar.module.css

```diff
@@ -0,0 +1,37 @@
+.defaultSubmit {
+  display: none;
+}
+
+.submitButton {
+  transform: translateX(-2rem);
+  margin-top: 0.25rem;
+  position: absolute;
+}
+
+.textInput {
+  width: 90vw;
```

Comment:
oof, I think this is the only dynamic unit that I wouldn't recommend using :) 
because it ignores sizes of its parent elements

Reply:
Would you recommend 90% instead? :)